### ISSUE TITLE
Multi-monitor Environment - problem with different resolution

### ISSUE THREAD

**Issue by Laurent27Normandie** [qgis/QGIS#47298]
### What is the bug or the crash?

Problem while working with a MacBook and external monitor. The same file is ok on my external screen but wrong on the laptop screen (as if it was a SCR Problem). Example : https://youtu.be/1lLYUTSh3u0. 

### Steps to reproduce the issue

Open the file on the external monitor
Open the same file without external monitor 
The projection is the same but seems being different...

### Versions

3.22.3-Białowieża

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [X] I tried with a new QGIS profile

**Comment by gioman** [comment]
@Laurent27Normandie @matematto same as https://github.com/qgis/QGIS/issues/46882 ?

**Comment by matematto** [comment]
exactly the same issue showed in the youtube video above
https://youtu.be/1lLYUTSh3u0

**Comment by Laurent27Normandie** [comment]
@gioman Yes it seems being exactly the same ;)

**Comment by gioman** [comment]
Closing as duplicate.

**Comment by alecameselle** [comment]
I'm so sorry to ask here again about an issue that seems solved, but I'm running into the same multi-monitor environment issue with my Macbook pro Built-in Retina Display 15.4-inch (2880x1800) and my HP 27xw Display 27-inch external monitor (1920x180). The layers of the QGIS project look well aligned when I view them on the HP monitor, but when I use the application on my Macbook pro's screen the raster layers are offset from the vector layers.

I have tried to follow the threads of issues #46882 and #46515 #46931, but I have not been able to solve the problem. I have limited knowledge of operating systems and I don't know where or how to change the variables QT_AUTO_SCREEN_SCALE_FACTOR = 0
QT_SCREEN_SCALE_FACTOR = 2.

Could someone tell me more precisely how to proceed?
